lefthook.yml

```diff
@@ -39,8 +39,8 @@ pre-push:
   parallel: true
   commands:
     typecheck:
-      run: pnpm turbo run typecheck
+      run: turbo run typecheck
```

Comment:
I assumed so, which is why I changed it.

In any case it seems that it was working for most, and this might be the change that has caused it not to work for some, so I think it would be safe to revert and investigate later.